examples/ReduxExample/README.md

```diff
@@ -1,5 +1,220 @@
-# Redux example
```

Comment:
Probably we should get rid of this generated README and use a simpler README saying how to run the project,